docs/SER4DATA.CSV

```diff
@@ -0,0 +1,521 @@
+Fri Dec 31 15:50:51 2021, 00:14:04, Recent State, 48, 21608, 12196, -67, 0, 38.84, 19.395, 0, 17.93, 1, 17.473, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, OFF, fwd, false, DRIVER_RUNNING, SPEED, 0, 0, [STATUS] ok., OFF, OFF, 0: 0010-1110 | 0000-0111 | 1: 1111-1111 | 1111-1111, 
```

Comment:
Good idea to add sample data, especially for writing the analysis scripts. Should we move this to a subfolder in data?